drivers/bridge/port_mapping.go

```diff
@@ -69,9 +83,15 @@ func (n *bridgeNetwork) allocatePort(bnd *types.PortBinding, containerIP, defHos
 		return err
 	}
 
+	portmapper := n.portMapper
+
+	if containerIP.To4() == nil {
```

Comment:
hoping we can be consistent with the way we decipher ipv6 and ipv4 throughout the codebase

Reply:
all checks should now check against `net.IP.To4()` so it should now be more consistent